### ISSUE TITLE
Introductory example outdated

### ISSUE THREAD

**Issue by jmtatsch** [chazzu/hass-animated-scenes#2]
Hello,

first I would like to thank you for providing this scene animation module.

I have some trouble to get it to work properly here, this may be due because your code has evolved faster than the provided example code.

```yaml
switch
- platform: animated_scenes
  name: Red Lights # name of scene
  restore: False # don't restore lights to previous state after scene, defaults to True
  ignore_off: False # if a light in the list is off, turn it on. Defaults to True, meaning lights will be ignored if they are off.
  lights: # list of light entities to modify
  - light.hue_color_lamp_1
  - colors:
    color_type: rgb # use rgb colors
    color: [255, 0, 0]
    brightness: 255 # maximum brightness
    weight: 2 # optional; higher number makes a given color more likely to appear. Default is 10, so you can choose some numbers to be less frequent
    one_change_per_tick: False # optional; defaults to false; only change color OR brightness on each tick, don't do both
  - colors:
    color_type: ct # color temperature colors
    color: 500 # number in mireds
  transition: 2 # take 2 seconds to transition
  change_frequency: 15 # change lights every 15 seconds
  change_amount: all # change all lights every time
  animate_brightness: True # defaults to true to change brightness level
  animate_color: True # defaults to true to change color
```

First it seems to me that this   '- colors:' fails to feed the expected list of colors
Second thing is color type ct does not seem to exist anymore.
Third color: 500 is not allowed by the input verification.
I think its should be more like:
``` yaml
    colors:
      - color_type: rgb_color # use rgb colors
        color: [255, 0, 0]
        brightness: 255 # maximum brightness
        weight: 2 # optional; higher number makes a given color more likely to appear. Default is 10, so you can choose some numbers to be less frequent
        one_change_per_tick: False # optional; defaults to false; only change color OR brightness on each tick, don't do both
      - color_type: color_temp # color temperature colors
        color: 250 # number in mireds
```

Unfortunately even with these changes that get me past input verification, it seems to only do one transition for me.

Could you please verify/update the example code and make sure it works as expected on your side?

**Comment by jmtatsch** [comment]
Actually the Red Light sample does work then, there was just too little difference for me to notice.
Input verification caps color for color_temp at 250 mired, but HA internal color definition seems to limit at 500, so input verification should be fixed.

Want me to whip up a pull request to fix these things?

**Comment by chazzu** [comment]
Apologies, I am paying attention to this project I just took a vacation from emails for the holidays. If you'd like to create a PR, I'd be more than happy to review and merge it. Alternatively, I will definitely get around to looking into this myself if needed. Based on a feature request for the ability to specify color lists per-light, I'll be working in this area anyway shortly.

**Comment by chazzu** [comment]
Resolved in c5cb1f9d9cda9340660cd9e2df66360c79584de2
